platform/database/database.go

```diff
@@ -30,7 +39,35 @@ var DefaultConfig = Config{
 	Database: "postgres",
 }
 
-func Connect(config Config) (*sqlx.DB, error) {
+var DefaultMetrics = Metrics{
+	QueryLatencyMs:  metrics.Float64("sql_query_latency_ms", "The query latency in milliseconds", metrics.UnitMilliseconds),
+	QueryErrorTotal: metrics.Float64("sql_query_error_total", "The query error total", metrics.UnitDimensionless),
+}
+
+var MetricViews = []*metrics.View{
+	&metrics.View{
+		Name:        "sql/query_latency",
+		Measure:     DefaultMetrics.QueryLatencyMs,
+		Description: "The distribution of the latencies",
+		Aggregation: metrics.ViewDistribution(0, 25, 100, 200, 400, 800, 10000),
```

Comment:
Not sure to understand the set of numbers here. 🤔 

Do we want to group queries based on latency, with each number as a category?

Reply:
yes. It generate a [cumulative histogram](https://prometheus.io/docs/practices/histograms/)